apis/sprints.py

```diff
@@ -156,7 +156,7 @@ def get_prompt_leaderboard(id, run_id):
 
     # Then query the leaderboard (ew)
     query = '''
-    SELECT run_id, path, runs.user_id, username, TIMESTAMPDIFF(MICROSECOND, runs.start_time, runs.end_time) AS run_time
+    SELECT run_id, path, runs.user_id, username, TIMESTAMPDIFF(MICROSECOND, runs.start_time, runs.end_time) AS run_time, runs.end_time AS end_time
```

Comment:
minor: any reason to explicitly use end time instead of the start time? I think it makes more sense to use the start time as the date you played the prompt

Reply:
Also, could you upload a video or pictures to show how this currently looks like, would be useful for UI suggestions and stuff

Reply:
No particular reason why I chose end_time, we can switch to start_time instead. 

As for the demo, could use populate_db.py for displaying runs for more than 1 test user lmao :))))) @dqian3

Reply:
Yeah I think start time makes more sense, but not a big deal